plugins/micromega/certificate.ml

```diff
@@ -985,7 +985,7 @@ let xlia_simplex env red sys =
     Prf (compile_proof env prf)
   in
   try
-    let sys = red sys in
+    (*let sys = red sys in*)
```

Comment:
If this code is no longer needed, it should just be removed. Or you may add a few sentences (e.g. borrowed from the text of your PR) to explain why calling `red` is no longer necessary.

Reply:
Will do! Still unsure it won't break... Happy the CI passed. Running bench...